src/index.tsx

```diff
@@ -1,19 +1,9 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import { DarkModeContextProvider } from "./contexts/DarkModeContext";
 
 const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
+  document.getElementById("root") as HTMLElement
 );
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+root.render(<DarkModeContextProvider><App /></DarkModeContextProvider>);
```

Comment:
przerzuciłbym Provider do App.tsx - masz tam wszystkie inne providery i warto to trzymać w jednym miesjcu - będzie ciut czytelniej